### ISSUE TITLE
False positive for prefer-array-literal when using `Array` from other namespace

### ISSUE THREAD

**Issue by NateScarlet** [microsoft/tslint-microsoft-contrib#856]
### Bug Report

-   **`tslint-microsoft-contrib` version**: 6.1.0-beta2
-   **TSLint version**: 5.12.1
-   **TypeScript version**: 3.3.3333
-   **Running TSLint via**:  VSCode

#### TypeScript code being linted

```tsx
import { Types } from 'mongoose';

export const foo: Types.Array<number> = new Types.Array();
```

with `tslint.json` configuration:

```json
  "extends": ["tslint-microsoft-contrib"],
  "rulesDirectory": ["node_modules/tslint-microsoft-contrib"],
  "rules": {
    "trailing-comma": false,
    "no-relative-imports": false,
    "no-implicit-dependencies": [true, ["dev", "graphql-tools"]],
    "newline-per-chained-call": false,
    "no-console": false,
    "quotemark": false,
    "export-name": [true, { "allow": ["resolvers"] }],
    "no-backbone-get-set-outside-model": false,
    "no-unsafe-any": false,
    "strict-boolean-expressions": false,
    "typedef": [
      true,
      "call-signature",
      "arrow-call-signature",
      "parameter",
      // "arrow-parameter",
      "property-declaration",
      "variable-declaration",
      "member-variable-declaration",
      "object-destructuring",
      "array-destructuring"
    ],
    "match-default-export-name": false
  }
```

#### Actual behavior

Got `Replace Array constructor with an array literal: new Types.Array() (prefer-array-literal)tslint(prefer-array-literal)`

#### Expected behavior

Got nothing

**Comment by JoshuaKGoldberg** [comment]
Good find @NateScarlet! Looks like the node should not be complained on if its `.paren` is a `ts.PropertyAccessExpression`.
